src/app/screens/weekly-exercises/index.tsx

```diff
@@ -0,0 +1,122 @@
+import { ArrowRightCircle } from 'lucide-react-native';
+import { useTheme } from 'styled-components';
+import React, { FC, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { FlatList, TouchableOpacity } from 'react-native';
+import { DrawerScreenProps } from '@react-navigation/drawer';
+
+import { Heading, ScreenContainer, Text } from 'src/app/components';
+import { AccordionItem } from 'src/app/components/accordion';
+import { MainDrawerParamList } from 'src/app/navigation/types';
+import { Exercise, MuscleGroup } from 'src/interfaces/exercises';
+import weeklyExercisesDataFile from 'src/mocks/weekly-exercises-data.json';
+
+import { ExerciseRow } from './styles';
+
+type Props = DrawerScreenProps<MainDrawerParamList, 'WeeklyExercises'>;
+
+export const WeeklyExercisesScreen: FC<Props> = ({ navigation }) => {
+  const { t } = useTranslation();
+  const theme = useTheme();
+  const [weeklyExercisesData, setWeeklyExercisesData] = useState<
+    Exercise[] | null
+  >(null);
```

Comment:
This comment is not required for the change, but you can initialize the value with an empty array instead of null